querybook/webapp/components/AIAssistant/QueryCellTitle.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useCallback, useState } from 'react';
```

Comment:
can you move this component to QueryCellTitle/QueryCellTitle? it is not AIAssistant specific